ReactNativeClient/lib/models/Tag.js

```diff
@@ -86,9 +175,57 @@ class Tag extends BaseItem {
 		});
 	}
 
-	static loadWithCount(tagId) {
-		const sql = 'SELECT * FROM tags_with_note_count WHERE id = ?';
-		return this.modelSelectOne(sql, [tagId]);
+	static async _addNoteCounts(tags) {
+		for (const tag of tags) {
+			const noteIds = await Tag.noteIds(tag.id);
+			// Make sure the notes exist
+			const notes = await Note.byIds(noteIds);
+			tag.note_count = notes.length;
```

Comment:
I think this is the same issue as the "full_title" property, and there should be a separate function to get the note count so as not to modify database rows.